services/src/datasets/external/gbif.rs

```diff
@@ -969,12 +970,49 @@ impl MetaDataProvider<OgrSourceDataset, VectorResultDescriptor, VectorQueryRecta
                     source: Box::new(e),
                 })?;
 
+        let conn = self
+            .pool
+            .get()
+            .await
+            .map_err(|e| Bb8Postgres { source: e })?;
+
+        let stats = conn
+            .query_one(
+                &format!(
+                    r#"
+                SELECT
+                    lower(time) AS tmin,
+                    upper(time) AS tmax,
+                    public.ST_XMIN(extent) AS xmin,
+                    public.ST_XMAX(extent) AS xmax,
+                    public.ST_YMIN(extent) AS ymin,
+                    public.ST_YMAX(extent) AS ymax
+                FROM {schema}.{taxonrank}_stats
+                WHERE name = '{canonicalname}'"#,
+                    schema = self.db_config.schema
+                ),
+                &[],
+            )
+            .await
+            .map_err(|e| Postgres { source: e })?;
+
+        let tmin = stats.get::<usize, NaiveDateTime>(0);
+        let tmax = stats.get::<usize, NaiveDateTime>(1);
+        let xmin = stats.get::<usize, f64>(2);
+        let xmax = stats.get::<usize, f64>(3);
+        let ymin = stats.get::<usize, f64>(4);
+        let ymax = stats.get::<usize, f64>(5);
+
         Ok(Box::new(StaticMetaData {
             loading_info: OgrSourceDataset {
                 file_name: self.db_config.ogr_pg_config().into(),
                 layer_name: format!("{}.{}", self.db_config.schema, self.occurrence_table),
                 data_type: Some(VectorDataType::MultiPoint),
-                time: OgrSourceDatasetTimeType::None, // TODO
+                time: OgrSourceDatasetTimeType::Start {
+                    start_field: "eventdate".to_string(),
+                    start_format: OgrSourceTimeFormat::Auto,
+                    duration: OgrSourceDurationSpec::Zero,
```

Comment:
If they want to see all occurrences in a certain month/year they should query a time range of this month/year. Is this not how it's expected to work?

Reply:
sure, the users could do that. I am only asking because previously we enlarged the temporal validity of occurrences to the whole year, because that makes it easier to compare it with other layers. Otherwise users will have to work with time shifts to e.g. see all occurrences from a certain year over the temperature of a single month.

On default, VAT selects a time instant and not a range, so you will most likely never see a single occurrence when you add GBIF data to the map unless you change to a time interval.